airflow/config_templates/config.yml

```diff
@@ -256,6 +256,13 @@
       type: integer
       example: ~
       default: "2"
+    - name: dagbag_import_tree_depth
+      description: |
+        If dags are defined in a subfolder, how deep will a DagFileProcessor look
```

Comment:
perhaps mention that zero, the default, means look in root folder. which i assume makes it backward compatible.